src/hooks/pager.ts

```diff
@@ -8,39 +8,60 @@ import { useService } from './service';
 
 export interface PagerManager {
     loadNext: () => void;
+    loadPrevious: () => void;
+    loadPage: (page: number) => void;
     reload: () => void;
     hasNext: boolean;
+    hasPrevious: boolean;
+    currentPage: number;
 }
 
 export function usePager<T extends AidboxResource>(
     resourceType: T['resourceType'],
     resourcesOnPage: number = 15,
     searchParams: SearchParams = {}
 ): [RemoteData<Bundle<T>>, PagerManager] {
-    const [pageToLoad, setPageToLoad] = useState(1);
+    const [pageToLoad, setPageToLoad] = useState(searchParams._page ? Number(searchParams._page) : 1);
```

Comment:
Should `searchParams._page` already be a number here? Can we use nullish coalescing operator instead?

```suggestion
    const [pageToLoad, setPageToLoad] = useState(searchParams._page ?? 1);
```